esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -312,20 +315,13 @@ DATABASE_NAME = '$DBNAME'
 DATABASE_HOST = 'localhost'
 DATABASE_PORT = '5432'
 
+VARNISH_HOST = 'localhost'
+VARNISH_PORT = '80'
+
 from database_settings import *
 
 MIDDLEWARE_LOCAL = []
-
-# E-mails for contact form
-email_choices = (
-    ('general', 'General Inquiries'),
-    ('web',     'Web Site Problems'),
-    )
-# Corresponding email addresses                                                                                                                                 
-email_addresses = {
-    'general': '$GROUPEMAIL',
-    'web':     '$SITENAME-websupport@learningu.org',
-    }
+SECRET_KEY = '`${CURDIR}/random_password.sh`'
```

Comment:
Ideally, we would have one (separate) file containing a template for settings.py.  The problem is that the different uses of settings.py files have different requirements: I think the Fabric setup uses a Python string template, whereas the new_site.sh script is using Bash substitution, and the Travis one is hardcoded.  I can try to reconcile these if you think that's in scope for this pull request.